zsl/application/service_application.py

```diff
@@ -29,9 +26,9 @@ def __init__(self, import_name, static_path=None, static_url_path=None,
 
     def initialize_dependencies(self, initialization_context):
         from zsl.application.initializers import injection_views, injection_modules
-        self._flask_injector = FlaskInjector(injection_views, injection_modules)
         self._initialization_context = initialization_context
-        self.set_injector(self._flask_injector.init_app(self))
+        self._flask_injector = FlaskInjector(self, injection_views + injection_modules)
+        self.set_injector(self._flask_injector.injector)
```

Comment:
I had to move FlaskInjector construction after setting of the initialization_context, because UnitTestInitializer was calling get_initialization_context within the FlaskInjector construction.